### PR TITLE
GH-264: Xpect files cannot be launched in IDE under Java 9

### PR DESCRIPTION
**Pull request by mor-n4**
This PR contains the exact same changes as PR #265. All discussion / review happened there.

This PR was only created to fix issues with the eclipse/eca checker.

### CODE REVIEW

**Comment by cdietrich** [comment]
please note: you should wait for the CQ to be approved before merging.

File: org.eclipse.xpect/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -25,7 +25,8 @@ Require-Bundle: org.apache.log4j;bundle-version="1.2.0";visibility:=reexport,
  org.eclipse.xtext.ecore;bundle-version="2.9.2";resolution:=optional,
  org.objectweb.asm;bundle-version="[3.0.0,8.0.0)";resolution:=optional,
  org.eclipse.equinox.common;bundle-version="3.0.0";resolution:=optional,
- org.eclipse.core.runtime;bundle-version="3.0.0";resolution:=optional
```
Comment (cdietrich) [inline org.eclipse.xpect/META-INF/MANIFEST.MF]: i wonder if this should be added to the pom as dependency too for standalone calling from maven/gradle

Comment (cdietrich) [reply org.eclipse.xpect/META-INF/MANIFEST.MF]: @dhuebner how do the maven artifacts you published deal with that issue? do they assume asm, equinox and others come from somewhere else/are added by the user?

Comment (mor-n4) [reply org.eclipse.xpect/META-INF/MANIFEST.MF]: When using maven, isn't this handled by tycho?

Comment (cdietrich) [reply org.eclipse.xpect/META-INF/MANIFEST.MF]: I just sag that the type id still eclipse bundle so the User has to take care about That itself (Same as for mwe)

Comment (szarnekow) [reply org.eclipse.xpect/META-INF/MANIFEST.MF]: Sorry I didn't get that.

Comment (cdietrich) [reply org.eclipse.xpect/META-INF/MANIFEST.MF]: the maven artifact type is eclipse-bundle => if you use pure gradle or maven then you have to care about transtive dependencies yourself as we do it for mwe

Comment (cdietrich) [reply org.eclipse.xpect/META-INF/MANIFEST.MF]: in mwe we do this. but i saw its not done in xpect in maven: https://github.com/eclipse/mwe/blob/9adfa2372a1d0d562734fbc8d9679357730b029e/plugins/org.eclipse.emf.mwe2.launch/pom.xml#L18

Comment (cdietrich) [reply org.eclipse.xpect/META-INF/MANIFEST.MF]: this is a general problem and not related to this specific pr. added a comment here: https://github.com/eclipse/Xpect/issues/267#issuecomment-504875500

**Comment by mmews-n4** [comment]
CQ 20234 was approved (see https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20234)

**Comment by mmews-n4** [comment]
Since both the CQ for the classgraph library was approved by the EMO team, and @meysholdt approved https://github.com/eclipse/Xpect/pull/265 which has identical changes compared to this PR, I understand this PR to be OK for merge now.

**Pull request merged**